draft-ietf-tls-tls13.md

```diff
@@ -1106,7 +1106,7 @@ and in TLS 1.3 the additional data input is empty (zero length).
 opaque_type
 : The outer opaque_type field of a TLSCiphertext record is always set to the
   value 23 (application_data) for outward compatibility with
-  middleboxes used to parsing previous versions of TLS.  The
```

Comment:
First Google hit for "used to": http://www.5minuteenglish.com/mar20.htm
In this context, "used to" is equivalent to "accustomed to".

The simplest change that would clarify this for those not familiar with this wording would be to just swap in "accustomed" for "used" with no other changes:

"middleboxes accustomed to parsing previous versions of TLS"